pyproject.toml

```diff
@@ -6,6 +6,7 @@ version = "0.1.0"
 dependencies = [
     "gpiozero",
     "pigpio",  # Run sudo pigpiod before running the program
+    "pytest",
```

Comment:
no, this is already specified in "optional dependencies" below, which is for developing

Reply:
I think while its not incorrect to put it within imu.py, it would cause the file to have 4 classes and be incredibly long, so I think its good to separate it. I like it being with logger and data_processor because both of them use it